sync/clients/sync.py

```diff
@@ -144,6 +144,20 @@ def get_project_recommendation(self, project_id: str, recommendation_id: str) ->
             )
         )
 
+    def get_project_submissions(self, project_id: str, params: dict = None) -> dict:
+        return self._send(
+            self._client.build_request(
+                "GET", f"/v1/projects/{project_id}/submissions", params=params
+            )
+        )
```

Comment:
This one doesn't get used (yet), but I included for parity with the `get_project_recommendations` method elsewhere in this file.